src/views/Sale/components/TokenFooter/index.tsx

```diff
@@ -138,6 +163,18 @@ export const TokenFooter: React.FC<TokenFooterProps> = ({ sale, saleDetails }: T
     mobileWrapper.width = 'calc(100% - 48px)'
   }
 
+  const addTokenToWallet = () => {
+    library?.send('wallet_watchAsset', {
+      type: 'ERC20',
+      options: {
+        address: sale?.tokenOut.id,
+        symbol: sale?.tokenOut.symbol,
+        decimals: sale?.tokenOut.decimals,
+        image: sale?.tokenOut.icon,
```

Comment:
Where do we store the icon?

Reply:
We have the `icon ` field in our Token interface, but its not getting populated at the moment from anywhere.